docs/docker-compose-keycloak.yml

```diff
@@ -22,7 +22,7 @@ services:
 
   gitlab:
     restart: always
-    image: sameersbn/gitlab:13.8.2
+    image: sameersbn/gitlab:13.9.0
```

Comment:
missing the minor update here...

Reply:
Sorry. I must have missed this file. I promise to do better next time.

Reply:
The files `contrib/docker-swarm/docker-compose.yml` and `docker-compose.swarm.yml` are also affected.

Reply:
fixed in https://github.com/sameersbn/docker-gitlab/pull/2319